x-pack/plugin/sql/qa/mixed-node/build.gradle

```diff
@@ -22,7 +22,7 @@ testClusters.configureEach {
 tasks.named("integTest").configure{ enabled = false}
 
 // A bug (https://github.com/elastic/elasticsearch/issues/68439) limits us to perform tests with versions from 7.10.3 onwards
-for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible.findAll { it.onOrAfter('7.10.0') }) {
+for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible.findAll { it.onOrAfter('7.10.3') }) {
```

Comment:
small nit but a comment would help on what changed in 7.10.3 vs 7.10.0 to make this version being considered.

Reply:
Using 7.10.0 was a bug. As the comment on line 24 says the version should always have been 7.10.3. But accidentally, the tests also worked with 7.10.0 until I introduced missing_order.